tox.ini

```diff
@@ -1,10 +1,34 @@
 [tox]
-envlist = py27, py34, py35
+minversion = 2.0
+envlist = py35,py34,py27,pep8
+skipsdist = True
 
 [testenv]
-deps=-r{toxinidir}/requirements.txt
-     -r{toxinidir}/test-requirements.txt
-       
-commands=
+usedevelop = True
+install_command = pip install -U {opts} {packages}
+setenv =
+   VIRTUAL_ENV={envdir}
+deps = -r{toxinidir}/test-requirements.txt
+commands =
    nosetests \
       []
+
+[testenv:py27-functional]
+commands = {toxinidir}/scripts/kube-init.sh nosetests []
+
+[testenv:coverage]
+commands = nosetests --with-coverage --cover-package=kubernetes.config,kubernetes.watch --cover-tests
```

Comment:
I don't see code coverage comment from codecov-io. If we do the code coverage right, we should get such a comment. The comment look like this: https://github.com/kubernetes-incubator/client-python/pull/89#issuecomment-271661688